pkg/deploy/identity-provider/identity_provider.go

```diff
@@ -123,11 +128,8 @@ func SyncIdentityProviderToCluster(deployContext *deploy.DeployContext) (bool, e
 	if isOpenShift {
 		doInstallOpenShiftoAuthProvider := instance.Spec.Auth.OpenShiftoAuth
 		if doInstallOpenShiftoAuthProvider {
-			openShiftIdentityProviderStatus := instance.Status.OpenShiftoAuthProvisioned
-			if !openShiftIdentityProviderStatus {
-				if err := CreateIdentityProviderItems(deployContext, cheFlavor); err != nil {
-					return false, err
-				}
+			if err := CreateIdentityProviderItems(deployContext, cheFlavor); err != nil {
```

Comment:
pls, rename it to `Sync..` otherwise it looks like we create items on every reconcile loop